File: behavior/src/main/java/com/google/behavior/MainActivity.java
package com.google.behavior;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private Button btnClick;
    private TextView tvContent;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btnClick = (Button) findViewById(R.id.btn_click);
        tvContent = (TextView) findViewById(R.id.tv_content);


        btnClick.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                tvContent.scrollTo(10, 0);
            }
        });

    }
}

File: animation/src/main/java/com/jv/animation/MainActivity.kt
package com.jv.animation

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.jv.animation.activity.FragmentActivity
import com.jv.animation.activity.ViewActivity

import com.jv.animation.utils.StatusBarUtils
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity(), View.OnClickListener {

    override fun onClick(v: View?) {
        if (v != null) {

        }
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        StatusBarUtils.setStatusBar(this)
        setContentView(R.layout.activity_main)

        btn_fragmentAnimation!!.setOnClickListener({ startActivity(Intent(this@MainActivity, FragmentActivity::class.java)) })
        btn_viewAnimation!!.setOnClickListener({ startActivity(Intent(this@MainActivity, ViewActivity::class.java)) })
    }
}
